### PR TITLE
Prepare for release 10.3.0.

### PR DESCRIPTION
**Pull request by nikhilnanivadekar**

### CODE REVIEW

**Comment by nikhilnanivadekar** [comment]
Please do not merge this PR. It is WIP.

File: RELEASE_NOTE_DRAFT.md [hunk 1]
```diff
@@ -32,47 +102,47 @@ Acquiring Eclipse Collections
...
 ```
 
 ### Gradle
 
 ```groovy
-compile 'org.eclipse.collections:eclipse-collections-api:10.2.0'
-compile 'org.eclipse.collections:eclipse-collections:10.2.0'
-testCompile 'org.eclipse.collections:eclipse-collections-testutils:10.2.0'
-compile 'org.eclipse.collections:eclipse-collections-forkjoin:10.2.0'
+compile 'org.eclipse.collections:eclipse-collections-api:10.3.0'
```
Comment (donraab) [inline RELEASE_NOTE_DRAFT.md:131]: Should this be `implementation` instead of `compile`?

File: RELEASE_NOTE_DRAFT.md [hunk 2]
```diff
@@ -1,27 +1,97 @@
...
+-------------------------------
+* Added OSGi metadata.
+* Fixed OSGi metadata to have sun.misc be an optional import package.
+* Fixed project versions published to p2 repository.
 
 # Build Changes
 -----------------
-* Moved some maven configuration from .travis.yml to jvm.config.
-* Changed Maven plugin repository to use https.
+* Migrated to GitHub Actions.
```
Comment (prathasirisha) [inline RELEASE_NOTE_DRAFT.md:80]: A suggestion to rephrase this line: 
Migrated all software development workflows or builds to GitHub Actions.

File: RELEASE_NOTE_DRAFT.md [hunk 3]
```diff
@@ -1,27 +1,97 @@
...
+* Upgraded dependencies used by tests.
+* Upgraded CheckStyle to 8.34
+* Upgraded Tycho plugin to 1.7.0
+* Upgraded EBR plugin to 1.2.0.
+* Upgraded versions of various Maven plugins.
+
+# Note
+-------
+_We have taken all the measures to ensure all features are captured in the release notes. 
+However, release notes compilation is manual, so it is possible that a commit might be missed.
```
Comment (prathasirisha) [inline RELEASE_NOTE_DRAFT.md:93]: 👍

**Comment by nikhilnanivadekar** [comment]
@prathasirisha good point about README and the examples. They are updated after the binary is available in maven central. In this distributed world we can be at best eventually consistent 😄

File: RELEASE_NOTE_DRAFT.md [hunk 4]
```diff
@@ -1,27 +1,97 @@
-10.2.0
+10.3.0
 ====================
 
-This is the 10.2.0 minor release.
-This release is to provide new features requested by the community, bug fix for edge case issues, and reduce tech debt.
+This is the 10.3.0 minor release. Even though this is a minor release, by no means the release was minor. This release is feature packed and comprises of numerious contributions from many contributors.
+This release is to provide new features requested by the community, bug fixes, reduce tech debt, and improve OSGi integration improvements.
```
[the remark this answers is not in the document]
Comment (nikhilnanivadekar) [reply RELEASE_NOTE_DRAFT.md]: fixed!

File: RELEASE_NOTE_DRAFT.md (hunk 1 above)
Comment (nikhilnanivadekar) [reply RELEASE_NOTE_DRAFT.md]: fixed

File: RELEASE_NOTE_DRAFT.md [hunk 5]
```diff
@@ -1,27 +1,97 @@
-10.2.0
+10.3.0
 ====================
 
-This is the 10.2.0 minor release.
-This release is to provide new features requested by the community, bug fix for edge case issues, and reduce tech debt.
+This is the 10.3.0 minor release. Even though this is a minor release, by no means the release was minor. This release is feature packed and comprises of numerious contributions from many contributors.
```
[the remark this answers is not in the document]
Comment (nikhilnanivadekar) [reply RELEASE_NOTE_DRAFT.md]: fixed

File: RELEASE_NOTE_DRAFT.md [hunk 6]
```diff
@@ -1,27 +1,97 @@
...
+* Implemented Interval.fromToExclusive().
+* Implemented Triples.
+* Implemented missing methods in Multimap factory.
+* Implemented Comparators.byFunctionNullsFirst(), Comparators.byFunctionNullsLast().
+* Implemented Bags.ofOccurrences() and Bags.withOccurrences().
+* Added Hindi Translation of Eclipse Collections website.
 
 # Bug Fixes
 -----------------
 * Fixed size edge case issues in Interval and IntInterval.
```
[the remark this answers is not in the document]
Comment (nikhilnanivadekar) [reply RELEASE_NOTE_DRAFT.md]: good catch. fixed.

File: RELEASE_NOTE_DRAFT.md (hunk 2 above)
Comment (nikhilnanivadekar) [reply RELEASE_NOTE_DRAFT.md]: fixed

**Comment by nikhilnanivadekar** [comment]
@donraab can you please verify if changes are ok?

**Comment by nikhilnanivadekar** [comment]
Merging based on code review approvals

**Pull request merged**